clar2wasm/src/standard/standard.wat

```diff
@@ -1182,6 +1179,25 @@
         )
     )
 
+    ;;
+    ;; Logical Not implementation
+    ;;
+    (type $not_func (func (param i32) (result i32)))
```

Comment:
We've been giving the parameters and locals names to improve readability. For example, you can use `(param $in_bool i32)` or something here.

Reply:
Done